packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md

```diff
@@ -161,9 +176,9 @@ The following table provides a list of all allowed HTML tags and elements in cus
 
 ## Add translations
 
-**Note:** After a template is customized, other languages are only included when they are customized. If all customized translations are reset, then the 27 default translations are used.
+**Note:** After you customize a template in one language, you need to customize other languages if you want to include them. If you reset all customized translations, then Okta uses the 27 default translations.
 
-The default language is used when the end user’s locale doesn't match any email customizations. You can edit the templates through the pencil icon, but you can't delete the default language template.
+Okta uses the default language when the end user’s locale doesn't match any email customizations. You can edit the templates through the pencil icon, but you can't delete the default language template.
```

Comment:
Should we mention that the only way to delete a default email customization is by resetting to default?

Reply:
Yes. i'll add it. Thanks!